doc/sphinx/dotnet/index.md

```diff
@@ -26,8 +26,13 @@ dotnet run --project examples/SoundSpeed
 ```
 
 ```{note}
-For current information on the state of the .NET API, refer to GitHub issue
-[Cantera/enhancements#156](https://github.com/Cantera/enhancements/issues/156).
+For current information on the state of the .NET API, refer to
+GitHub issues and enhancement requests, specifically:
+
+- [**Open Pull Requests**](https://github.com/Cantera/cantera/pulls?q=is%3Apr+state%3Aopen+label%3A.net)
+- [**Open Issues**](https://github.com/Cantera/cantera/issues?q=is%3Aissue%20state%3Aopen%20label%3A.net)
+- [**Work-in-progress Trackers**](https://github.com/Cantera/enhancements/issues?q=is%3Aissue%20state%3Aopen%20label%3A.net%20label%3Awork-in-progress)
+- [**Feature-Requests**](https://github.com/Cantera/enhancements/issues?q=is%3Aissue%20state%3Aopen%20label%3A.net%20label%3Afeature-request)
```

Comment:
The number of these is so small (one each) that I'd be inclined to combine them under a single heading.